docs/src/pages/demos/buttons/FloatingActionButtonZoom.tsx

```diff
@@ -59,7 +59,7 @@ const useStyles = makeStyles((theme: Theme) =>
 
 function FloatingActionButtonZoom() {
   const classes = useStyles();
-  const theme = useTheme();
+  const theme = useTheme<Theme>();
```

Comment:
For imports from `@material-ui/core/styles` we should have the core Theme as the default type argument. If no  theme provider is in the tree then the default theme will be used.

This is not the case for `@material-ui/styles` which is why the type argument is required for those imports.